services/core-web/src/components/admin/AdminNavigation.tsx

```diff
@@ -1,17 +1,16 @@
-import React from "react";
-import PropTypes from "prop-types";
+import React, { FC } from "react";
 import { Link } from "react-router-dom";
 import { Menu } from "antd";
 import { DownOutlined } from "@ant-design/icons";
 import { includes } from "lodash";
 import * as routes from "@/constants/routes";
 
-const propTypes = {
-  activeButton: PropTypes.string.isRequired,
-  openSubMenuKey: PropTypes.arrayOf(PropTypes.string).isRequired,
+interface AdminNavigationProps {
+  activeButton: string,
+  openSubMenuKey: string[],
 };
 
-export const AdminNavigation = (props) => {
+const AdminNavigation: FC<AdminNavigationProps> = (props: AdminNavigationProps) => {
```

Comment:
FYI, once the FC<AdminNavigationProps> is there, it will infer the propType for props without explicitly typing it.